### ISSUE TITLE
Mention breaking change in AWSLambdaMigrateFunctionTask

### ISSUE THREAD

**Issue by musketyr** [JustinPihony/gradle-aws-plugin-reboot#14]
there is a breaking change when `timeout` has changed to `lambdaTimeout` without any warning notice

**Comment by JustinPihony** [comment]
You are correct - I will modify the release notes to signify the breaking change. It had to do with the Gradle 7.1 upgrade where  it conflicted with `getTimeout` added to `ConventionTask`

**Comment by JustinPihony** [comment]
Another option could be to use the timeout built into the `AbstractTask` (not sure why I didn't try it then) - I may look into that first and see if it is usable in the same fashion

**Comment by musketyr** [comment]
That's probably not the best idea. Let's say that someone would like to timeout the upload task itself because of for example bad connection then there will be no option how to set this separately.
